File: client/entity/DockerInfo.py
# https://blog.csdn.net/weixin_44191019/article/details/108377525
class DockerInfo:
    containerId = ""
    name = ""
    cpu = "0.0"
    memeryUsage = "0.0"
    memeryLimit = "0.0"
    memeryUsageRate = "0.0"
    netI0Send = '0.0MB'
    netIOReceive = '0.0MB'
    blockIORead = '0.0MB'
    blockIOWrite = '0.0MB'
    pids = 0

    # 构造函数
    def __init__(self, containerId, name, cpu, memeryUsage, memeryLimit, memeryUsageRate, netIOSend, netIOReceive,
                 blockIORead, blockIOWrite, pids):
        self.name = name
        self.containerId = containerId
        self.cpu = cpu
        self.memeryUsage = memeryUsage
        self.memeryLimit = memeryLimit
        self.memeryUsageRate = memeryUsageRate
        self.netIOSend = netIOSend
        self.netIOReceive = netIOReceive
        self.blockIORead = blockIORead
        self.blockIOWrite = blockIOWrite
        self.pids = pids

    def toString(self):
        return "containerId:" + self.containerId + ",name:" + self.name + ",cpu:" + self.cpu + ",memeryUsage:" + self.memeryUsage + ",memeryLimit:" + self.memeryLimit + ",memeryUsageRate:" + self.memeryUsageRate + ",netIOSend:" + self.netIOSend + ",netIOReceive:" + self.netIOReceive + ",blockIORead:" + self.blockIORead + ",blockIOWrite:" + self.blockIOWrite + ",pids:" + str(
            self.pids)

File: src/main/java/cn/luckynow/monitoringserver/entity/DockerInfo.java
package cn.luckynow.monitoringserver.entity;

import com.baomidou.mybatisplus.annotation.TableField;
import com.baomidou.mybatisplus.annotation.TableName;
import java.io.Serializable;
import java.sql.Timestamp;

import io.swagger.annotations.ApiModel;
import io.swagger.annotations.ApiModelProperty;
import lombok.Data;

/**
 * <p>
 * 
 * </p>
 *
 * @author xz
 * @since 2023-08-11
 */
@TableName("docker_info")
@ApiModel(value = "DockerInfo对象", description = "")
@Data
public class DockerInfo implements Serializable {

    private static final long serialVersionUID = 1L;

    private Long id;
    private String ip;
    private Timestamp date;

    @ApiModelProperty("容器名称")
    private String name;

    @ApiModelProperty("容器id")
    private String containerId;

    private String cpu;

    private String memeryUsage;

    private String memeryLimit;

    private String memeryUsageRate;

    @ApiModelProperty("网络发送量")
    @TableField("net_io_send")
    private String netIOSend;

    @ApiModelProperty("网络接收量")
    @TableField("net_io_receive")
    private String netIOReceive;

    @ApiModelProperty("磁盘读取量")
    @TableField("block_io_read")
    private String blockIORead;

    @ApiModelProperty("磁盘写入量")
    @TableField("block_io_write")
    private String blockIOWrite;


}
